cluster/images/hyperkube/Makefile

```diff
@@ -19,7 +19,7 @@
 
 REGISTRY?=gcr.io/google_containers
 ARCH?=amd64
-TEMP_DIR:=$(shell mktemp -d)
+TEMP_DIR=/tmp/hyperkube
```

Comment:
Regarding:

> *rename TEMP_DIR to BUILD_DIR. 

It makes sense considering your changes.

>*use another location .e.g. ./_output or ../../../_output/

Yeah, since we aren't creating temporary directories it also makes sense.